### PR TITLE
Setup base

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
- [x] add root files
- [x] add 'src' directory and populate
- [x] add 'test' directory and populate
- [x] add 'config' directory and populate
- [x] correctly name apps

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps @gsvr please review.

**Comment by NicolaasDieBaas** [comment]
@gsvr please review.

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+    grunt.initConfig({
+        paths: {
+            src: {
+                app: {
+                    ussd_clinic: 'src/ussd_clinic.js',
+                    ussd_chw: 'src/ussd_chw.js',
+                    ussd_public: 'src/ussd_public.js',
+                    ussd_optout: 'src/ussd_optout.js',
+                    sms_smsinbound: 'src/sms_smsinbound.js',
```
Comment (gsvr) [inline Gruntfile.js]: Let's just call this `sms_inbound`

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Copyright (c) Praekelt Foundation and individual contributors.
```
Comment (gsvr) [inline LICENSE]: I'm used to seeing a date here, like this: `Copyright (c) 2016, Praekelt Foundation`, not sure how important that is.

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+NDOH MomConnect Apps
```
Comment (gsvr) [inline README.rst:1]: Should this file rather be Markdown?

File: config/config_sms.nurse_sms.qa.json.example [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (gsvr) [inline config/config_sms.nurse_sms.qa.json.example]: These config file names don't match with the app names. But I personally don't think we should have these example configs at this time (or ever - they tend to just lag behind and therefore be unhelpful).

File: test/sms_nurse.test.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        var tester;
+
+        beforeEach(function() {
+            app = new go.app.GoNDOH();
+
+            tester = new AppTester(app);
+
+            tester
+                .setup.config.app({
+                    name: 'nurse_sms',
```
Comment (gsvr) [inline test/sms_nurse.test.js]: naming consistency

File: test/sms_nurse.test.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+                    env: 'test',
+                    testing: 'true',
+                    testing_today: 'June 30, 2017 09:07:07'
```
Comment (gsvr) [inline test/sms_nurse.test.js]: You could potentially remove these config values above at the moment if you want a very clean app.

**Comment by gsvr** [comment]
@NicolaasDieBaas Some comments, mostly about naming. :+1: when you've made the changes and Travis is happy.  Take note that Travis is not activated at the moment - I've asked for someone to switch it on - will let you know if I hear anything - usually you have to push some code after activation to get it to run for the first time.

File: test/sms_inbound.test.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        var tester;
+
+        beforeEach(function() {
+            app = new go.app.GoNDOH();
+
+            tester = new AppTester(app);
+
+            tester
+                .setup.config.app({
+                    name: "smsinbound",
```
Comment (gsvr) [inline test/sms_inbound.test.js:17]: sms_inbound

**Pull request merged**